packages/components/inputs/date-input/src/date-input.tsx

```diff
@@ -165,20 +256,20 @@ const DateInput = (props) => {
                   onMouseEnter: () => {
                     // we remove the highlight so that the user can use the
                     // arrow keys to move the cursor when hovering
-                    if (isOpen) setDownshiftHighlightedIndex(null);
+                    if (isOpen) setDownshiftHighlightedIndex(0);
```

Comment:
Is this correct? I think this should stay as `null`. See https://github.com/downshift-js/downshift/blob/1c7698080805e05f5180dd223cbc17932a85fd42/src/downshift.js#L80